### DISCUSSION TITLE
Draft Mode docs is missing an explanation of how to enable it

### DISCUSSION THREAD

**Discussion by alvarlagerlof** [vercel/community#3093] [Bug Report]
### Page to Investigate

https://vercel.com/docs/workflow-collaboration/draft-mode#using-draft-mode-with-sveltekit

### Steps to Reproduce

https://vercel.com/docs/workflow-collaboration/draft-mode#enabling-draft-mode-in-preview-deployments

This sections shows the Vercel toolbar with a purple background color, and links to [Preview Deployments](https://vercel.com/docs/concepts/deployments/preview-deployments), indicating that is the page where draft mode can be toggled via the toolbar. But that pages does not mention draft mode at all.

I do not get the "eye" icon whatsoever, even when `const { isEnabled } = draftMode();` returns true. I don't think I've ever seen it, either on pages/ or app/ Next.js sites. I persume that other than turning on Draft mode in Next.js, there must be some setting somewhere that I've missed?

Is this feature locked higher than the Pro plan, or are the docs wrong / missing something?

**Comment by S3Prototype** [comment]
Hi @alvarlagerlof! We just rolled out changes so that this feature should be available to **all** users. Could you visiting your preview deployment again? You should see the eye icon in your preview toolbar. If not, please let me know and we'll look into it ASAP

**Reply by alvarlagerlof** [reply]
Nothing. Draft mode being on/off doesn't seem to matter either.

<img width="279" alt="image" src="https://github.com/vercel/community/assets/14835120/ce482ef1-17a6-4f84-9cb1-6c42511a9e98">

**Reply by alvarlagerlof** [reply]
I also redeployed it and it didn't show up.

**Reply by S3Prototype** [reply]
Oh, that's unfortunate 😞 . Are you able to share:
1. The repository
2. The deployment URL with the full path to the page

If you're on a pro or enterprise account, you should [create a support ticket here](https://vercel.com/help#issues) to get help from an engineer instead 👍

**Reply by alvarlagerlof** [reply]
For sure, it's open source.

https://github.com/alvarlagerlof/portfolio/pull/995

https://portfolio-m8r5su5l3-lagerlof.vercel.app/api/draft
https://portfolio-m8r5su5l3-lagerlof.vercel.app/blog/skeleton-loading-with-suspense-in-next-js-13

**Reply by S3Prototype** [reply]
Okay, I think the hole in our docs is that you need to use ISR on the page for draft mode to work. Can you try adding ISR?

[Here's the Next.js docs on the subject](https://nextjs.org/docs/app/building-your-application/data-fetching/revalidating)

**Reply by alvarlagerlof** [reply]
I use the app router and `export const revalidate = ...` my site.

**Reply by S3Prototype** [reply]
Yes, I can see you are exporting `revalidate` now that I look. I'll report this to an engineer and get back to you.

By the way were you able to open a support ticket?

**Reply by S3Prototype** [reply]
Actually, it looks like you're exporting `revalidate` in `app/(backnav)/blog/[slug]/page.tsx`, but it appears you want to use `createClientWithDraftMode().fetch(query);` in `app/(main)/blog/page.tsx`.

Could you try exporting revalidate in `app/(main)/blog/page.tsx`? (assuming you haven't tried already)

**Reply by alvarlagerlof** [reply]
Isn't using draft mode and the eye icon a separate problem? I'm able to turn on draft mode in my code without the Vercel toolbar if I want to, but I'm still not seeing the eye icon. Why would those to be related other than the eye being able to turn on draft mode for me.

**Reply by S3Prototype** [reply]
One reason the eye icon may not be appearing is If you're not using revalidation in a route

**Reply by alvarlagerlof** [reply]
I do have `export const revalidate = 600;` in [app/(backnav)/blog/[slug]/page.tsx](https://github.com/alvarlagerlof/portfolio/blob/0a6498eef0616917d4f562d50911a97f4f567a93/app/(backnav)/blog/%5Bslug%5D/page.tsx#L8). Which was the page that I was testing. Shouldn't that be enough?
